assets/templates/search/bar.tmpl

```diff
@@ -0,0 +1,11 @@
+<div class="wrapper">
+    <form class="col-wrap search__form" action="/search">
+        <h1>Search our website</h1>
+        <label class="font-size--16 block" for="nav-search">Keyword(s) or time series ID</label>
+        <input type="search" autocomplete="off" class="search__input search__input--results-page col col--md-29 col--lg-29" id="nav-search" name="q" value="">
```

Comment:
We'd probably want this input to contain the the users search string (e.g. value of parameter `q`)